diff -Naur a/integration-cli/docker_cli_save_load_unix_test.go b/integration-cli/docker_cli_save_load_unix_test.go
--- a/integration-cli/docker_cli_save_load_unix_test.go	2015-07-13 20:18:41.445366118 +0000
+++ b/integration-cli/docker_cli_save_load_unix_test.go	2015-07-13 20:20:03.261376016 +0000
@@ -8,7 +8,7 @@
 	"os"
 	"os/exec"
 
-	"github.com/docker/docker/vendor/src/github.com/kr/pty"
+	"github.com/kr/pty"
 	"github.com/go-check/check"
 )
 
