diff -Naur a/api/server/server.go b/api/server/server.go
--- a/api/server/server.go	2015-07-13 20:20:51.381381716 +0000
+++ b/api/server/server.go	2015-07-13 20:21:56.733389316 +0000
@@ -13,8 +13,8 @@
 	"strings"
 	"time"
 
-	"code.google.com/p/go.net/websocket"
 	"github.com/gorilla/mux"
+	"golang.org/x/net/websocket"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api"
diff -Naur a/integration-cli/docker_api_attach_test.go b/integration-cli/docker_api_attach_test.go
--- a/integration-cli/docker_api_attach_test.go	2015-07-13 20:20:51.381381716 +0000
+++ b/integration-cli/docker_api_attach_test.go	2015-07-13 20:22:32.721393433 +0000
@@ -9,7 +9,7 @@
 
 	"github.com/go-check/check"
 
-	"code.google.com/p/go.net/websocket"
+	"golang.org/x/net/websocket"
 )
 
 func (s *DockerSuite) TestGetContainersAttachWebsocket(c *check.C) {
